src/windows.rs

```diff
@@ -24,5 +24,13 @@ pub fn load_native_certs() -> Result<Vec<Certificate>, Error> {
         }
     }
 
+    if let Ok(current_machine_store) = schannel::cert_store::CertStore::open_local_machine("ROOT") {
```

Comment:
Why are we not propagating any error from the `CertStore::open_local_machine()` call?

It would also be nice to avoid duplicating the core loop. Perhaps we can construct a `Vec` for `CertStore` values and have an outer loop iterating over it?